### PR TITLE
[platform_channels] adds EventChannel implementation for iOS

### PR DESCRIPTION
**Pull request by AyushBherwani1998**
## Description

Adds EventChannel implementation for iOS to listen the Accelerometer updates. Improves the error message for event channel demo.

### CODE REVIEW

File: platform_channels/ios/Flutter/.last_build_id [hunk 1]
```diff
@@ -1 +1 @@
-dad366ac47fa5a1d30c8533697937d44
\ No newline at end of file
+7787d82f894e9250d86178e1b6b3e78d
```
Comment (RedBrogdon) [inline platform_channels/ios/Flutter/.last_build_id:1]: I checked with @jmagman, and this file shouldn't be checked into source control. Do you mind deleting/gitignoring it?

File: platform_channels/ios/Runner/AccelerometerStreamHandler.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import CoreMotion
+
+class AccelerometerStreamHandelr: NSObject, FlutterStreamHandler {
```
Comment (RedBrogdon) [inline platform_channels/ios/Runner/AccelerometerStreamHandler.swift:7]: nit: Class name is misspelled (should end with "Handler").

**Review by RedBrogdon — CHANGES_REQUESTED** [review]
LGTM, with two small nits.

File: platform_channels/ios/Runner/AccelerometerStreamHandler.swift (hunk 2 above)
Comment (AyushBherwani1998) [reply platform_channels/ios/Runner/AccelerometerStreamHandler.swift]: Done :D

File: platform_channels/ios/Flutter/.last_build_id (hunk 1 above)
Comment (AyushBherwani1998) [reply platform_channels/ios/Flutter/.last_build_id]: Done :D

**Review by RedBrogdon — APPROVED** [review]
LGTM. Thanks for rolling with my holiday-induced delay. 🦃

**Comment by AyushBherwani1998** [comment]
@RedBrogdon Hope you had a good time. Finally, only one thing left for the platform channel samples, and we can finally remove `This sample is currently being built.` from README 🙈

**Pull request merged**